src/sklearn_evaluation/plot/pca.py

```diff
@@ -114,7 +115,8 @@ def _plot_generic(n_components, principal_components, ax):
     for pc1 in range(n_components):
         for pc2 in range(pc1 + 1, n_components):
             ax[ax_ind].scatter(
-                principal_components[:, pc1], principal_components[:, pc2], s=50
+                principal_components[:, pc1], principal_components[:, pc2],
+                alpha=.6, edgecolor="face", s=50
```

Comment:
same as my previous comment. is it better to set the edgecolor property inline rather than at the apply_theme level?

Reply:
According to [matplotlib api](https://matplotlib.org/stable/tutorials/introductory/customizing.html#the-default-matplotlibrc-file) docs it's possible using these attributes:

```
#scatter.marker: o         # The default marker type for scatter plots.
#scatter.edgecolors: face  # The default edge colors for scatter plots.
```

I tried
```
plt.rcParams['scatter.edgecolors'] = 'face'
```

But for some reason, it doesn't work. btw, changing `scatter.marker` works.

Reply:
alright, if it doesn't work, let's harcode it then